### DISCUSSION TITLE
Colored dot in front of group candidates

### DISCUSSION THREAD

**Discussion by patrickli** [Aloxaf/fzf-tab#269] [Q&A] [answered]
I'm not sure if this is a bug so opened a discussion instead of an issue.

When presenting group candidates, where is an extra "dot" with group color in front of the values. You can see this in multiple screenshots other people posted (for other purposes).

![example 1](https://user-images.githubusercontent.com/15178513/137610672-7fead6b9-24b2-4b6d-8475-22a3362a452a.png)
![example 2](https://user-images.githubusercontent.com/17017672/72494883-3d9df780-3860-11ea-971b-fc3987289407.png)
![example 3](https://user-images.githubusercontent.com/15178513/143974865-d8fb6626-96e4-48cf-8ec9-9a11b4b29e01.png)

I wonder is this intentional or something that can be turned off?

**Comment by deadnews** [comment]
It's fzf.

Fix: https://github.com/junegunn/fzf/commit/2157f4f1933ac289976f42a4e347f444857ba18f

> `fzf --color gutter:-1`

---

Wiki: https://github.com/junegunn/fzf/wiki/Color-schemes

```sh
export FZF_DEFAULT_OPTS=$FZF_DEFAULT_OPTS'
--color=gutter:-1
'
```

My conf:

```sh
export FZF_DEFAULT_OPTS=$FZF_DEFAULT_OPTS'
--color=gutter:-1,bg+:#4f5666
'
```

**Reply by patrickli** [reply]
I don't think that's the gutter. The gutter is the column the pointer resides. If set to `--color=gutter:-1` it merely turns off the background color of the pointer column.

**Comment by patrickli** [comment, marked as answer]
Well I guess I just need to RTFM. https://github.com/Aloxaf/fzf-tab/wiki/Configuration#prefix
